libkbfs/interfaces.go

```diff
@@ -1246,6 +1246,9 @@ type Prefetcher interface {
 	// CancelPrefetch notifies the prefetcher that a prefetch should be
 	// canceled.
 	CancelPrefetch(kbfsblock.ID)
+	// RemovePrefetch notifies the prefetcher that a prefetch should be
```

Comment:
If you're going to keep this function (even though it's not used anymore), maybe explain here the difference between remove and cancel, like you did in the comments?